app/database/migrations/20210213121940_create_clients_table.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable('clients', table => {
+        table.integer('job').unsigned().notNullable();
+        table.string('first_name');
+        table.string('last_name');
+        table.string('email', 320);
```

Comment:
Was there a reason for choosing 320 as the size and not more or less? I believe 255 is the default size too.

Reply:
320 is the maximum length of an email address.

Reply:
you can just use text if you want